pkg/apis/core/types.go

```diff
@@ -486,6 +486,24 @@ const (
 	PersistentVolumeClaimFileSystemResizePending PersistentVolumeClaimConditionType = "FileSystemResizePending"
 )
 
+type PersistentVolumeClaimResizeStatus string
```

Comment:
Should we also update comments on the PVC.spec.capacity field with the new feature-gated behavior for shrinking?

Reply:
Updated